lib/app/features/posts/data/favorites_repository.dart

```diff
@@ -17,7 +16,7 @@ class FavoritesRepository {
     try {
       return database.insertFavoritePost(FavoritePostsCompanion.insert(post: post));
     } on Exception {
```

Comment:
@alexdivad please change this try so it has a exception with a toString so you get more accurate info